data-access-analysis-tools/index.rst

```diff
@@ -36,9 +36,19 @@ Scroll down to the bottom of this page for more information about the future RSP
 Rubin Science Platform (RSP)
 ============================
 
-The Rubin Science Platform (RSP) provides access to Rubin Observatory data products.
-The :ref:`Tools-RSP-Notebook` enables programmatic interaction with, and analysis of, data products in a python environment.
-The :ref:`Tools-RSP-Portal` provides interactive query, data discovery, and visualization tools.
+The Rubin Science Platform (RSP) provides access to Rubin Observatory data products via three aspects: Portal, Notebooks, and API.
```

Comment:
Does API need to be spelled out? Same comment for other acronyms below such as ADQL and TOPCAT.

Reply:
all acronyms spelled out, as this is their first appearance on the page